### PR TITLE
Refactor/dom sleep

### PR DESCRIPTION
**Pull request by e-spitz**
# Description

Display all sleep data on the DOM using API data.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring

# How Has This Been Tested?
localhost8080

- [x] open index.html
- [x] dev tools
- [ ] terminal

# Checklist:

- [x] My code follows the Turing's guidelines of this project
- [x] I have performed a self-review of my own code
- [x] No console error messages

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -200,22 +207,31 @@ const renderDailyUserGlasses = (user, userRepo) => {
 
 const renderAllUserDailyOz = (userRepo) => {
   hydrationFriendOuncesToday.innerText = userRepo.calculateAverageDailyWater(todayDate);
+  renderBestWorstSleep(userRepo);
 }
 
-// user.findFriendsTotalStepsForWeek(userRepository.users, todayDate);
+const renderUserSleepToday = (user) => {
```
Comment (EricSergeant) [inline src/scripts.js:213]: Good refactor!

**Review by EricSergeant — APPROVED** [review]
Good completion of sleep and cleanup of old code.

File: src/scripts.js [hunk 2]
```diff
@@ -8,12 +8,11 @@ import Activity from './Activity';
...
 
-let user;
-let users;
+let user, users;
+let userRepository = new UserRepository();
 let sleep, sleepData;
 let hydrationData;
-let userRepository = new UserRepository();
-let todayDate = "2020/01/22";
+let todayDate = "2020/01/22"; //fix to be dynamic
```
Comment (dorifuto-dev) [inline src/scripts.js:15]: Good job setting a global variable todayDate.

**Review by dorifuto-dev — APPROVED** [review]
Looks great!

**Pull request merged**